velox/core/QueryConfig.h

```diff
@@ -549,6 +553,13 @@ class QueryConfig {
     return get<uint8_t>(kSpillStartPartitionBit, kDefaultStartBit);
   }
 
+  /// Returns the timestamp unit used in Velox-Arrow conversion.
+  /// 0: second, 1: milli, 2: micro, 3: nano.
```

Comment:
how about define `0: second, 3: milli, 6: micro, 9: nano`, means the number of digits of fractional seconds

Reply:
Both should work. Maybe continous configuarable values are more common.

Reply:
I'm not sure which system you're referring to with "continuous configurable values" for timestamp precision. AFAIK, in the timestamp syntax for MySQL, PostgreSQL, and Flink, the precision is indicated by the number of digits of fractional seconds.

Reply:
I'm trying to say if we use discrete numbers, it may be confusing to user what happens if setting a value between. But I do not have strong opinion here. Both should work.

Reply:
Understood. Theoretically, Velox could accommodate any precision between 0 and 9, but since Arrow only has three common precisions, such a configuration makes sense if it's solely for Arrow. However, IMHO, if we were to interface with another system that only supports millisecond and nanosecond, or supports any arbitrary precision like MySQL, this method of configuration could lead to confusion.

Reply:
Updated. Thanks.